src/SprykerEco/Zed/ComputopApi/Business/Mapper/ExpressCheckout/PayPalExpressPrepareMapper.php

```diff
@@ -15,15 +15,15 @@ class PayPalExpressPrepareMapper extends AbstractPayPalExpressMapper
     /**
      * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
      *
-     * @return array<string, string>
+     * @return array<string, int|string>
      */
     protected function encryptRequestData(QuoteTransfer $quoteTransfer): array
     {
-        $computopPayPalExpressPaymentTransfer = $quoteTransfer->getPayment()->getComputopPayPalExpress();
+        $computopPayPalExpressPaymentTransfer = $quoteTransfer->getPaymentOrFail()->getComputopPayPalExpressOrFail();
 
         return [
-            ComputopApiConstants::DATA => $computopPayPalExpressPaymentTransfer->getData(),
-            ComputopApiConstants::LENGTH => $computopPayPalExpressPaymentTransfer->getLen(),
+            ComputopApiConstants::DATA => $computopPayPalExpressPaymentTransfer->getDataOrFail(),
```

Comment:
these fields were not required before
please do not break backward compatibility

Reply:
it's okay to have `getOrFail` while chaining
but the last call should be a simple `get` in this case

Reply:
Fixed